sdcm/fill_db_data.py

```diff
@@ -1114,7 +1114,7 @@ class FillDatabaseData(ClusterTester):
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 2,
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 3,
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 4,
-                        "#REMOTER_SUDO nodetool flush",
```

Comment:
I would recommend splitting this change from the workaround to downgrade JDK, once you proven this to be working.

Reply:
worked [here](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/5.0/job/rolling-upgrade-ubuntu20.04-test-jdk-downgrade/2/)

i will split it to another commit then, 1 for the workaround and 1 for this change (proved that it worked)